src/Login/LoginForm.tsx

```diff
@@ -77,6 +78,29 @@ export const LoginForm: React.FC<LoginFormProps> = () => {
                 setLoginAlertMsg({ message: err.message, variant: 'danger' });
             }
         }
+
+        try {
+            const res = await AxiosRequest.get('/users/status');
+            const data = res.data.data as { userPaidUntil: Date, universityPaidUntil: Date };
+            const userPaidMoment = data.userPaidUntil.toMoment();
+            const universityPaidMoment = data.universityPaidUntil.toMoment();
+
+            let paidUntil: moment.Moment | undefined;
+            let accountType: AccountType | undefined;
+            if (userPaidMoment.isAfter(universityPaidMoment)) {
+                paidUntil = userPaidMoment;
+                accountType = AccountType.INDIVIDUAL;
+            } else {
+                paidUntil = universityPaidMoment;
```

Comment:
The paid until field on the user is the source of truth, in theory, it should always be equal to or after the university's paid until but I don't want to impose that restriction

If we ever wanted to block a professor for a semester for whatever reason we could reduce their paid until and until the university paid their account would be blocked